emulator/src/main/java/org/wso2/carbon/protocol/emulator/dsl/AbstractProtocolEmulator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * *
+ *  * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
most of the time I see licences without this extra "*" mark, I feel it's better if we follow same here.